src/main/java/cinema/model/dto/MovieSessionResponseDto.java

```diff
@@ -0,0 +1,49 @@
+package cinema.model.dto;
+
+public class MovieSessionResponseDto {
+    private Long id;
+    private Long movieId;
+    private String movieTitle;
```

Comment:
I think we should either return Id of entity, or fields of this entity we think is OK to expose. Not both